gammapy/spectrum/tests/test_reflected.py

```diff
@@ -161,16 +164,17 @@ def bad_on_region(exclusion_mask, on_region):
 
 
 @requires_data()
-class TestReflectedRegionBackgroundEstimator:
-    def test_basic(self, bkg_estimator):
-        assert "ReflectedRegionsBackgroundEstimator" in str(bkg_estimator)
-
-    def test_run(self, bkg_estimator):
-        assert len(bkg_estimator.result[1].off_region) == 11
-        assert "Reflected" in str(bkg_estimator.result[1])
-
-    @requires_dependency("matplotlib")
-    def test_plot(self, bkg_estimator):
-        with mpl_plot_check():
-            bkg_estimator.plot(idx=1, add_legend=True)
-            bkg_estimator.plot(idx=[0, 1])
+def test_reflected_bkg_maker(spectrum_dataset_maker, reflected_bkg_maker, observations):
+
```

Comment:
Remove empty line at start of functions (consistency within Gammapy)

Reply:
Done